src/handlebars/archive.handlebars

```diff
@@ -60,16 +70,16 @@
                           \{{else}}
                             <td></td>
                           \{{/if}}
-                          <td class='download-td'>
-                            <a class='blue-button no-underline' href='\{{link}}'><var binary-info>\{{type}} (\{{size}} MB)</var></a>
-                          </td>
                           \{{#if installer_link}}
                           <td class='download-td'>
-                            <a class='blue-button no-underline' href='\{{installer_link}}'><var binary-info>Installer \{{type}}</var></a>
+                            <a class='blue-button no-underline' href='\{{installer_link}}'><var binary-info>\{{type}} (\{{installer_extension}})</var></a>
                           </td>
                           \{{else}}
-                          <td></td>
+                          <td><div class='empty-download'>&nbsp;</div></td>
                           \{{/if}}
+                          <td class='download-td'>
+                            <a class='grey-button no-underline' href='\{{link}}'><var binary-info>\{{type}} (\{{size}} MB)</var></a>
+                          </td>
```

Comment:
Future work would be good to move the metadata (size, type, checksum, etc) out to a separate info panel for this download to avoid clutter on the archive page.